libseccomp/src/lib.rs

```diff
@@ -1624,7 +1624,11 @@ impl ScmpFilterContext {
 
     /// Gets a raw pointer of a seccomp filter.
     ///
-    /// This function return a raw pointer to the [`scmp_filter_ctx`].
+    /// This function returns a raw pointer to the [`scmp_filter_ctx`].
+    /// The caller must ensure that the filter outlives the pointer this function returns,
+    /// or else it will end up pointing to garbage.
+    /// In addition, modifying the filter referenced by the pointer will cause an invalid
+    /// filter.
```

Comment:
Modifying with `libseccomp_sys::seccommp_*()` should be fine, it don't?

Reply:
Yes, I'll fix it as follows. WDYT?

>/// In addition, modifying **directly** the filter referenced by the pointer will cause
>/// an invalid filter.

Reply:
> In addition, you may only modify the filter with functions intended for this (the once provided by libseccomp_sys).

Reply:
Thank you. I fixed it.